image/verify/fuzz/Cargo.toml

```diff
@@ -0,0 +1,54 @@
+# Licensed under the Apache-2.0 license
+
+[package]
+name = "caliptra-image-verify-fuzz"
+version = "0.0.0"
+publish = false
+edition = "2021"
+
+[package.metadata]
+cargo-fuzz = true
+
+[dependencies]
+libfuzzer-sys = { version = "0.4.6", optional = true }
+afl = { version = "0.13.3", optional = true }
+
+[patch.crates-io]
+byteorder = { git = "https://github.com/benjamindoron/byteorder.git", branch = "struct_aware-1.4.3" }
+zerocopy = { git = "https://github.com/benjamindoron/zerocopy.git", branch = "struct_aware-v0.6.3" }
+
+[dependencies.caliptra-lms-types]
+path = "../../../lms-types"
+features = ["arbitrary"]
+
+[dependencies.caliptra-image-types]
+path = "../../types"
+features = ["arbitrary"]
+
+[dependencies.caliptra-drivers]
+path = "../../../drivers"
+
+[dependencies.caliptra-image-verify]
+path = ".."
+
+[features]
+struct-aware = []
+
+# Prevent this from interfering with workspaces
```

Comment:
As this is not part of the workspace, we need some sort of CI step that will make sure this project builds / is formatted correctly.

Reply:
https://github.com/chipsalliance/caliptra-sw/pull/631 will run nightly, but that's not really soon enough. Maybe a for-loop (in the main build CI workflow) over each target to run these common steps will do.

Reply:
Thanks @benjamindoron, a for loop over these targets to run cargo build/fmt/clippy works for us

Reply:
Okay, I'll push that to a separate PR, since it's not specific to any existing at the moment.

Note that clippy is broken until https://github.com/chipsalliance/caliptra-sw/issues/681 is done. The patch mechanism does not support features, so, I derived from `arbitrary` without checking. But that depends on `std`, which is usually disabled in those crates we used.

Reply:
Done, see https://github.com/chipsalliance/caliptra-sw/pull/700.